### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by ItsVops** [LunarClient/BukkitAPI#33]
Saw something about this is a different issue, but that was a while ago.

I can't seem to find any updated documentation about the API?

**Comment by OskarZyg** [comment]
Javadoc jars are published on the maven repo iirc

**Comment by ItsVops** [comment]
Yeah, but it seems as many methods and packets are outdated as they were for the old lunar client.
